### ISSUE TITLE
Assertion failed

### ISSUE THREAD

**Issue by hjpotter92** [lassik/emacs-format-all-the-code#186]
I'm initialising the mode as follows:

```emacs-lisp
(use-package format-all
  :bind
  (("<f8>" . format-all-buffer)
   ("<f9>" . format-all-mode))
  :custom
  ((format-all-formatters (
                           ("Python" black)
                           ("CSS" prettier)
                           ("HTML" html-tidy))))
  :delight
  :hook
  ((format-all-mode . format-all-ensure-formatter)
   (prog-mode . format-all-mode)))
```

I keep getting the following error when opening any file (not necessarily python/css/html):

```
Eager macro-expansion failure: (cl-assertion-failed (equal (mapcar 'car body) '(:executable :install :languages :features :format))) [2 times]
or: Assertion failed: (equal (mapcar 'car body) '(:executable :install :languages :features :format))
Eager macro-expansion failure: (cl-assertion-failed (equal (mapcar 'car body) '(:executable :install :languages :features :format))) [2 times]
File mode specification error: (cl-assertion-failed (equal (mapcar 'car body) '(:executable :install :languages :features :format)))
```

**Comment by meain** [comment]
Duplicate of #184

**Comment by ve-nt** [comment]
Apply #185 to your local copy to fix this issue for youself, at least until its merged and a new version is shipped.

**Comment by lassik** [comment]
Fixed by #185
